bootstrap/kubeadm/config/webhook/manifests_Validating.yaml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
might be adding a comment explaining why are we creating this (those) files

Reply:
I deleted them. We don't need these files actually. They were created for a previous commit, but forgot to delete them.